test/run.sh

```diff
@@ -62,10 +62,8 @@ if [ ! -f "${RF_VARIABLES}" ]; then
     exit 1
 fi
 
-# shellcheck disable=SC2034
-DEST_DIR="${RF_VENV}" 
-"${ROOTDIR}/scripts/fetch_tools.sh" robotframework
-"${ROOTDIR}/scripts/fetch_tools.sh" yq
+DEST_DIR="${RF_VENV}" "${ROOTDIR}/scripts/fetch_tools.sh" robotframework
+DEST_DIR="${RF_VENV}" "${ROOTDIR}/scripts/fetch_tools.sh" yq
```

Comment:
Why is it necessary to change the default DEST_DIR?

Reply:
Because `DEST_DIR` specifies where to install the tool. robotframework is installed in `microshift/_output/robotenv` instead of `microshift/microshift-rf-tests@tmp/venv`. This causes robot framework execution to fail: `/home/jenkins/ws/workspace/microshift/microshift-rf-tests/microshift/test/run.sh: line 104: /home/jenkins/ws/workspace/microshift/microshift-rf-tests@tmp/venv/bin/robot: No such file or directory`

I found this bug last night Jenkins run: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/microshift/job/microshift-rf-tests/923/console

This change PR reverts the change in the [Stress Testing PR](https://github.com/openshift/microshift/pull/2647/files#diff-608b6425c35171211ed108a6c30a5121be49f5f8f7bfdab4df44414377f26dd0L64)

Reply:
The reason there's a variable is so if there is environment-specific need it can be handled by the caller of this script. Can we make the installation destination use the default here, and export the variable in the QE script that invokes it?